lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/ActiveContextBridgeFactory.java

```diff
@@ -35,7 +35,7 @@
 @ThreadSafe
 @Private
 public final class ActiveContextBridgeFactory {
```

Comment:
The changes in this file are now redundant with the ones merged as REEF-328, right? If so, please remove them.